### ISSUE TITLE
File manager's upload file overwriting feature is labelled in a confusing way, and never prompts the user

### ISSUE THREAD

**Issue by nekohayo** [webmin/webmin#2113]
In the File Manager's "Advanced options", there is a setting called "Do not overwrite existing files upon operations", where the default is "No" and "Always ask".

There are a couple of issues with this:

* **It is a double-negation,** so it's hard to understand, it's confusing.  
It would better to flip everything around and name this something like "Overwrite files when uploading", and the "No" would become either become "Yes" or "Keep the same name and overwrite", and the previously-named "Yes" choice would become "Name the uploaded file differently (add a number at the end)"… or something like that.

* **The "Always ask" checkbox is unclear,** and does not seem to work.  
Presumably it means that if enabled, it would prompt the user to ask them what to do or to ask for confirmation before overwriting... however as you can see in the attached video here, it does not work:

[Webmin file manager upload behavior inconsistent, no overwriting prompts.webm](https://github.com/webmin/webmin/assets/479401/79e05975-a27e-4139-8e38-7e283244540a)

Tested with Firefox 124 on Webmin 2.105 / Virtualmin 7.10.0 running on Debian 12.

**Comment by jcameron** [comment]
@iliajie I think this is a theme-controlled option?

**Comment by nekohayo** [comment]
Alternate possibility, which would be a cleaner/simpler UX: don't make this a setting/option, just prompt me to make a choice between "Keep the same names and overwrite files" and "Rename uploaded files to avoid conflicting" everytime I start an upload that has conflicting filenames; I believe this approach would make more sense, because this type of choice is probably highly context-dependent, more than a "global preference" kind of thing…

**Comment by iliajie** [comment]
> * **It is a double-negation,** so it's hard to understand, it's confusing.

Agreed, I will call it **Preserve existing files upon operations** instead.

> * name this something like "Overwrite files when uploading"

This feature ins't about uploading files only. 


> Presumably it means that if enabled, it would prompt the user to ask them what to do or to ask for confirmation before overwriting

Yes, but not for uploads. For other operations, like when extracting a file:

<img width="1179" alt="image" src="https://github.com/webmin/webmin/assets/4426533/1afea4e2-1cc3-4f8d-9294-674ee2746f8b">

> Alternate possibility, which would be a cleaner/simpler UX: don't make this a setting/option, just prompt me to make a choice between

Sorry, that wouldn't work for a lot of other folks.

> Everytime I start an upload that has conflicting filenames; I believe this approach would make more sense

In particular, in application to uploads, it isn't easy to implement; so, it's much simpler to ask before the process.
